assets/prometheus-k8s/role-specific-namespaces.yaml

```diff
@@ -112,4 +112,20 @@ items:
     - get
     - list
     - watch
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: prometheus-k8s
+    namespace: openshift-userworkload-monitoring
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - services
+    - endpoints
+    - pods
+    verbs:
+    - get
+    - list
+    - watch
```

Comment:
oops, those should not end up in this asset, just in the userworkload prometheus.

Reply:
correction: this indeed is correct, we need this to scrape the `openshift-userworkload-monitoring` namespace.